code/modules/hydroponics/plants_herb.dm

```diff
@@ -149,15 +149,10 @@ ABSTRACT_TYPE(/datum/plant/herb)
 
 	New()
 		. = ..()
-		START_TRACKING_CAT(TR_CAT_CANNABIS_OBJ_ITEMS)
```

Comment:
?

Reply:
Ah crap, I must have accidentally undid this while removing any other testing on the patrol behavior (i used cannabis as the patrol beacons, easier to move than actual beacons)